float32[] state_ref
float32[] state_asv
float32 des_ang
float32 e_ang
float32 v_ref
float32 e_v
float32 u_rudder
float32 u_thrust
float32 I_thrust
float32 I_rudder
string action
